pkg/client/group_mgmt_client.go

```diff
@@ -232,10 +232,13 @@ func (client *GroupMgmtClient) listGetOrPost(path string, params *param.GetParam
 			fetch := "fetch"
 			wrapper := &DataWrapper{
 				Data:          params.Filter,
-				StartRow:      params.Page.StartRow,
-				EndRow:        params.Page.EndRow,
 				OperationType: &fetch,
 			}
+			if params.Page != nil && params.Page.StartRow != nil && params.Page.EndRow != nil {
```

Comment:
Isn't it enough to just check params.Page != nil?

Reply:
No, since we can have nil for StartRow /EndRow.

Reply:
Aren't wrapper.StartRow and wrapper.EndRow initialized to nil anyway?  Why would it hurt to set them to nil?  It seems better to have fewer conditions.

Reply:
Its better to have this check otherwise it may lead to a crash if you try to  access nil value since it doesn't exists. 
Its fine .

Reply:
How could a crash occur?

Consider a client program that sets params.Page.StartRow and not params.Page.EndRow.  With this implementation, we will silently drop the specified StartRow value.  By removing the extra conditions, the StartRow will be passed along to the server, which might opt to return an error if no EndRow value is given.

Reply:
We can go either way. I believe having this extra check or not . I didn't want to have an explicit nil value to be set for wrapper.StartRow if param.Page.StarRow is not set. Anyways i will make this change.